schemata/base.yaml

```diff
@@ -28,3 +28,114 @@ definitions:
     uniqueItems: true
     example:
     - "the-domains/the-grid"
+  geo:
+    type: object
+    description: Geographical data
+    properties:
+      latitude:
+        name: latitude
+        type: number
+        description: Latitude coordinate
+        example: 45.5
+      longitude:
+        name: longitude
+        type: number
+        description: Longitude coordinate
+        example: -122.7
+      zoom:
+        name: zoom
+        type: number
+        description: Zoom level of map
+        example: 4.0
+  bbox:
+    type: object
+    properties:
+      x:
+        name: x
+        type: number
+        description: "Cartesian coordinate along x-axis"
+        example: 608.1907
+      y:
+        name: y
+        type: number
+        description: "Cartesian coordinate along y-axis"
+        example: 189.9090
+      width:
+        name: width
+        type: number
+        description: "Width"
+        example: 229.2087
+      height:
+        name: height
+        type: number
+        description: "Height"
+        example: 229.2087
+  tuplecolor:
```

Comment:
Better to call `colorarray` or even `rgbarray` ? `description should mention this is using 8 bit integer colors `[0 255]` and not `[0.0 1.0]`.